cmd/cut-release/main.go

```diff
@@ -94,18 +97,26 @@ func generateTypescriptBrowserDocs() {
 
 // Update the version string in all files that must be updated for a new release
 func updateHardCodedVersions(version string) {
-
-	// Update `rpc/clients/typescript/package.json`
-	tsClientPackageJSONPath := "rpc/clients/typescript/package.json"
+	// Update `packages/rpc-client/package.json`
+	tsClientPackageJSONPath := "packages/rpc-client/package.json"
 	newVersionString := fmt.Sprintf(`"version": "%s"`, version)
 	regex := `"version": "(.*)"`
 	updateFileWithRegex(tsClientPackageJSONPath, regex, newVersionString)
 
-	// Update `browser/package.json`
-	browserPackageJSONPath := "browser/package.json"
+	// Update `packages/browser-lite/package.json`
+	browserLitePackageJSONPath := "packages/browser-lite/package.json"
+	newVersionString = fmt.Sprintf(`"version": "%s"`, version)
+	regex = `"version": "(.*)"`
+	updateFileWithRegex(browserLitePackageJSONPath, regex, newVersionString)
+
+	// Update `packages/browser/package.json`
+	browserPackageJSONPath := "packages/browser/package.json"
 	newVersionString = fmt.Sprintf(`"version": "%s"`, version)
 	regex = `"version": "(.*)"`
 	updateFileWithRegex(browserPackageJSONPath, regex, newVersionString)
+	newBrowserLiteDependencyString := fmt.Sprintf(`"@0x/mesh-browser-lite": "^%s"`, version)
+	regex = `"@0x/mesh-browser-lite": "(.*)"`
```

Comment:
In retrospect, this is pretty subtle, and I'll add a comment in my cut-release fix PR.